### PR TITLE
Fix grammar and increase consistency

### PR DESCRIPTION
**Pull request by BurkHufnagel**
Modified the README file to increase the consistency of things like how 'Note' is used, start sentences with a capital letter, and make things read a bit more smoothly -- IMO.

Added modifications based on the emailed comments I received. One thing to note -- Markdown seems to get unhappy after a total of three starting asterisks, so I removed the inner ones around "failing" because they didn't seem to be doing anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ _Project(s) without tests_ often end up looking like they are stuck together wit
 
 ![duct tape car fail](http://i.imgur.com/9cNriGK.jpg)
 
-Change _one_ part and the _other_ stops working? "_Fixing_" one bug, creates another?
+Change _one_ part and _another_ stops working? "_Fixing_" one bug, creates another?
```
Comment (nelsonic) [inline README.md]: https://dictionary.cambridge.org/dictionary/english/another 👍

File: README.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ _Project(s) without tests_ often end up looking like they are stuck together wit
...
 
-Wouldn't you *prefer* if everything was
+Wouldn't you *prefer* it if everything was
```
Comment (nelsonic) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -1045,7 +1043,7 @@ Refresh the Code Coverage report in your browser:
 
 ![server-side-test-istanbul-coverage-report](images/server-side-test-istanbul-coverage-report.png)
 
-> _**Boom**_! You know how to run your QUnit-based Unit Tests server-side!
+> _**Boom**_! Now you know how to run your QUnit-based Unit Tests on the server-side!
```
Comment (iteles) [inline README.md]: We have a confluence of two expressions here now which will need to be changed ➡️  either `server-side!` or `on the server!` 
Both work and `server-side` is more common in tech circles but if you think it's not as accessible to a beginner @BurkHufnagel then `on the server` is cool 👍

File: README.md [hunk 4]
```diff
@@ -1144,10 +1142,10 @@ This _clearly_ documents the functionality of the `getChange` method.
 
 In the last **90 minutes** you _**learned how**_ to:
 + Write code following **T**est **D**riven **D**evelopment (**TDD**) discipline
-+ View the **code coverage** for both front-end and back-end JavaScript Code
++ Generate and view the **code coverage** for both front-end and back-end JavaScript Code
 + Set up **Travis-CI Continuous Integration** for your project
 (so that you can keep track of the test/build status for your project)
-+ Use **JSDoc** to document your code using simple comment blocks
++ Use **JSDoc** to generate documentation for your code after writing simple comment blocks
```
Comment (iteles) [inline README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -212,7 +212,7 @@ Now copy-paste the following *sample code* to get started:
 #### Open index.html in your Browser
 
 When you ***open*** `index.html` in your ***web browser***
-you should expect to see something like this: (_without the annotations_)
+you should expect to see something like this: (_without the green annotations_)
```
Comment (iteles) [inline README.md]: The annotations here are both green (on passing tests) and red (on failing tests) so we'll need to change this sentence either back to what it was or to be more explicit on the green and red annotations (not to be confused with QUnit's own green and red defaults) so as not to be misleading ☺️

File: README.md [hunk 6]
```diff
@@ -31,34 +31,34 @@ Test Driven Development (TDD) makes your team a well-oiled machine which means y
 Once you have a ***suite*** of tests that run on every change, you will
 begin to develop a whole other level of ***confidence*** in your codebase
 and will discover a new freedom to be ***creative*** without fear of
-"*breaking*" anything else; truly *game-changing*.
+"*breaking*" anything unexpectedly; truly *game-changing*.
```
Comment (nelsonic) [inline README.md]: 👍

File: README.md [hunk 7]
```diff
@@ -31,34 +31,34 @@ Test Driven Development (TDD) makes your team a well-oiled machine which means y
...
 
 
 ## What?
 
 This tutorial will help you get started with
 **T**est **D**riven **D**evelopment (**TDD**) *today*! <br />
 In the next ***30 minutes*** you will learn _everything_<sup>1</sup>
-you need to know to write tests for your web project!
+you need to write tests for your web project!
```
Comment (nelsonic) [inline README.md]: the "know" was included to emphasise the removal of the knowledge barrier. 🤔 
but I agree that the sentence can be simplified. 👍

File: README.md [hunk 8]
```diff
@@ -79,22 +79,22 @@ is that writing code following TDD (*discipline*)
...
-What this means is that there's a **3-step process**:
+What this means is that TDD follows a **3-step process**:
 
-1. ***Write*** a **Failing Test** - Understand the (user)
+1. ***Write a Failing Test** - Understand the (user)
 requirements/story well enough to write a test for what you expect.
 (_the test should **fail** initially - hence it being "Red"_) <br />
 
-2. ***Make*** the (*failing*) **Test Pass** - Write (*only*) the code you need
+2. ***Make the (*failing*) Test Pass** - Write (*only*) the code you need
```
Comment (nelsonic) [inline README.md]: this markdown will fail because the opening `***` is closed by `**` ...

File: README.md [hunk 9]
```diff
@@ -79,22 +79,22 @@ is that writing code following TDD (*discipline*)
...
 to make the (*failing*) test pass, while ensuring your existing/previous tests
 all still pass (*no regressions*).
 
-3. ***Refactor*** the code *you* wrote - if you have time to tidy up the code
+3. ***Refactor*** the code *you* wrote - take the time to tidy up the code
```
Comment (nelsonic) [inline README.md]: definite improvement. 👍

File: README.md [hunk 10]
```diff
@@ -127,30 +127,30 @@ even when you're working with a large, inter-dependent code base!
...
-we encourage you to follow our _complete_ Todo List Tutorial:
-[https://github.com/dwyl/**todo-list-javascript-tutorial**](https://github.com/dwyl/todo-list-javascript-tutorial)
-Which is a step-by-step guide to building an App
-following testing and documentation-first best practices.
+so you can focus on the principals of testing.
+Once you understand the basics,
+we encourage you to follow our _complete_ Todo List Tutorial 
+([https://github.com/dwyl/**todo-list-javascript-tutorial**](https://github.com/dwyl/todo-list-javascript-tutorial)),
+which is a step-by-step guide to building an App
+following testing and documentation-first best practices._
```
Comment (nelsonic) [inline README.md]: this rewording is much better. thanks. 👍

File: README.md [hunk 11]
```diff
@@ -246,7 +246,7 @@ and loading the CSS files.
 
 Next we see the **qunit divs**
 (_where the **test results** will be **displayed**_)
-and load JQuery and the QUnit Library from CDN:
+and load the JQuery and QUnit Libraries from CDN:
```
Comment (nelsonic) [inline README.md]: 👍

**Pull request merged**